gemini-app/src/DataContext.js

```diff
@@ -198,7 +199,7 @@ export const DataProvider = ({ children }) => {
     const [selectedSensorQuery, setSelectedSensorQuery] = useState(null);
 
     // Backend
-    const [flaskUrl, setFlaskUrl] = useState("http://127.0.0.1:5050/flask_app/");
+    const [flaskUrl, setFlaskUrl] = useState("http://127.0.0.1:5053/flask_app/");
```

Comment:
Let's keep the default as 5050. You might need to remove this from your changes.